cilium/cmd/ip_list.go

```diff
@@ -80,27 +90,14 @@ func printEntry(w *tabwriter.Writer, entry *models.IPListEntry) {
 
 	ni := identity.NumericIdentity(*entry.Identity)
 	identityNumeric := ni.StringID()
-	var identities []string
+	var labels []string
 	if numeric {
-		identities = append(identities, identityNumeric)
+		labels = append(labels, identityNumeric)
 	} else {
-		identity := ni.String()
-		if identity != identityNumeric {
-			identities = append(identities, identity)
-		} else {
-			params := ipApi.NewGetIdentityIDParams().WithID(identity).WithTimeout(api.ClientTimeout)
-			id, err := client.Policy.GetIdentityID(params)
-			if err != nil {
-				Fatalf("Cannot get identity for given ID %s: %s\n", id, err)
-			}
-			lbls := labels.NewLabelsFromModel(id.Payload.Labels)
-			for _, lbl := range lbls {
-				identities = append(identities, lbl.String())
-			}
-		}
+		labels = append(labels, getLabels(ni)...)
 	}
 	first := true
-	for _, identity := range identities {
+	for _, identity := range labels {
```

Comment:
nit: Since we've renamed identities-labels perhaps we should use label for the loop var for clarity?

Reply:
Fixed, good pointer :)